worldedit-core/src/main/java/com/sk89q/worldedit/function/mask/Mask.java

```diff
@@ -26,10 +26,12 @@
 /**
  * Tests whether a given vector meets a criteria.
  */
+@FunctionalInterface
 public interface Mask {
 
     /**
-     * Returns true if the criteria is met.
+     * Returns true if the criteria is met and
+     * placement is allowed for the block.
```

Comment:
Why was this change made? Masks do not necessarily only apply to blocks, I would leave this doc comment as is.

Reply:
To me the docs felt a bit unclear wheather it means "the criteria is met to allow" or "the criteria is met to block".
So it might be nice if the ambiguity could be stripped too, but I would leave the phrasing open to you then.

Reply:
It's neither. Masks can be used to block or allow. It's literally up to the usage of the mask object.

Reply:
At least the masks I've found so far seem to follow the **true=allow** convention.
So I assume manifesting this in the docs, would make the interface easier to use.
But not sure if any **true=deny** masks exist yet, so I leave the decision up you then.

Reply:
I would prefer just reverting this for now.

Reply:
Alright then.